src/sync-function-loader.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
The contents of this file are identical to the previous revision of `etc/sync-function-loader.js`.